rss_feed/forms.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.syndication.views import Feed
+from blog.models import BlogPage
+
+
+class BlogsFeed(Feed):
+    title = "Cos.io blog"
+    link = "/blogs-rss/"
```

Comment:
Does this mean that the RSS feed can be found at both `/blog/feed/` and `/blogs-rss/`? Or should this be the url that's registered in urls.py (i.e. `/blog/feed/`)?

Reply:
This link attribute is somewhat inexplicable, it appears in the RSS response and points back to the page that's sending it. The docs on this are pretty [vague](https://chrxr.com/adding-rss-feed-wagtail-site/)